src/components/value-list/value-list.tsx

```diff
@@ -457,6 +458,14 @@ export class ValueList<
     }
   };
 
+  @Listen("calciteValueListItemDragHandleBlur")
```

Comment:
I _think_ this can be achieved by using `focusout`, but let's proceed with this for now and please open a follow-up issue to revisit this approach.

Reply:
Added a follow-up issue https://github.com/Esri/calcite-components/issues/6526